dashboard/app/helpers/plc/evaluations_helper.rb

```diff
@@ -0,0 +1,34 @@
+module Plc::EvaluationsHelper
+  def determine_preferred_learning_modules level_source, script
```

Comment:
The clearest argument that this shouldn't live in a view helper is that it makes multiple `ActiveRecord` queries, bypassing the controller.

We have a general problem in our codebase that helpers have become a dumping ground for various logic that shouldn't live in the view and doesn't really make sense in the controller.  I've been advocating pushing this logic down to the models, and separating out Concerns (or Plain Old Ruby Objects) where appropriate.